src/pointers.cpp

```diff
@@ -202,7 +202,7 @@ namespace big
         // Trigger Script Event
         {
             "TSE",
-            "45 8B F0 41 8B F9 48 8B EA",
+            "45 8B F8 41 8B F9",
```

Comment:
This update is actually for SCRIPT::TRIGGER_SCRIPT_EVENT, not SCRIPT::SEND_TU_SCRIPT_EVENT